waspc/src/Wasp/AI/OpenAI/ChatGPT.hs

```diff
@@ -53,18 +53,17 @@ queryChatGPT apiKey params requestMessages = do
     Utils.HTTP.httpJSONThatThrowsIfNot2xx request
       <&> either (error . ("Failed to parse ChatGPT response body as JSON: " <>)) Prelude.id
 
-  when True $
-    pTrace
-      ( "\n\n\n\n==================================\n"
-          <> "\n===== GPT PARAMS ======\n"
-          <> show params
-          <> "\n====== REQUEST =======\n"
-          <> show requestMessages
-          <> "\n====== RESPONSE ======\n"
-          <> show chatResponse
-          <> "\n==================================\n\n\n\n"
-      )
-      $ return ()
+  debugTrace
```

Comment:
We had this printing all the time before. Now, I made it so that it prints only when DEBUG env vars is set.